primitives/transaction/src/account/staking_contract/structs.rs

```diff
@@ -176,10 +176,10 @@ impl IncomingStakingTransactionData {
                 verify_transaction_signature(transaction, proof)?
             }
             IncomingStakingTransactionData::AddStake { .. } => {
-                // Adding stake should be at least greater than 0.
-                if transaction.value.is_zero() {
-                    warn!("Add stake transactions must have positive value. The offending transaction is the following:\n{:?}", transaction);
-                    return Err(TransactionError::ZeroValue);
+                // Adding stake should be greater than 0.
+                if transaction.value < Coin::from_u64_unchecked(Policy::MINIMUM_STAKE) {
+                    warn!("Add stake must increment stake by at least minimum stake. The offending transaction is the following:\n{:?}", transaction);
+                    return Err(TransactionError::InvalidValue);
```

Comment:
Is this not a problem for older transactions?

Reply:
Yeah. This needs to still allow syncing txs that were valid before the upgrade.

Reply:
Correct, I'll fix it and push the changes
Thanks!